selenium/che-selenium-core/bin/webdriver.sh

```diff
@@ -428,7 +441,7 @@ printRunOptions() {
     echo "[TEST]"
     echo "[TEST] =========== RUN OPTIONS ==========================="
     echo "[TEST] Mode                : "${MODE}
-    echo "[TEST] Rerun failing tests : "${RERUN}
+    echo "[TEST] Rerun attempts      : "${RERUN_ATTEMPTS}
```

Comment:
Does that mean all tests will be rerun ?

Reply:
Not, it doesn't. I have just realised that the code will be simpler if we replace `rerun = false` by identical by meaning `rerun attempts = 0`.

Reply:
ok